README.md

```diff
@@ -90,6 +90,7 @@ define(['form'], function(form) {
     messages: {
       'required': 'Field required.',
       'min': 'Enter a value greater than or equal to {0}.',
+      'max': 'Please enter a value greater than or equal to {0}.',
       'maxlength': 'Enter a value with max length less than or equal to {0}.',
       'pattern': 'Enter a valid value.',
       'email': 'Enter a valid email address.'
```

Comment:
I think this example its not right, because you don't have to pass override messages that way any more.

@tisvasconcelos